gcc/cp/contracts.cc

```diff
@@ -1717,6 +1891,69 @@ build_contract_violation (tree contract, contract_continuation cmode)
   return ctor;
 }
 
+static int
+get_contract_assertion_kind(tree contract)
+{
+  switch (TREE_CODE (contract))
+  {
+    case ASSERTION_STMT:	return CAK_ASSERT;
+    case PRECONDITION_STMT:	return CAK_PRE;
+    case POSTCONDITION_STMT:	return CAK_POST;
+    default:
+      error ("Invalid contract kind");
+  }
+
+  return CAK_INVALID;
+}
+
+static int
+get_evaluation_semantic(tree contract, contract_continuation cmode)
+{
+  contract_semantic semantic = get_contract_semantic (contract);
+
+  if (checked_contract_p(semantic))
+    {
```

Comment:
you don't need these braces (and in other similar cases)

Reply:
removed the ones i could spot